worlds/metroidfusion/data/locations.py

```diff
@@ -1024,7 +1024,10 @@ class Sector6AfterVariaCoreXZone(FusionRegion):
 ]
 
 Sector4Hub.locations = [
-    FusionLocation("Sector 4 (AQA) -- Drain Pipe", False, [Requirement([CanDrainAQA], [HasWaveBeam, CanDefeatMediumGeron]),
+    FusionLocation("Sector 4 (AQA) -- Drain Pipe", False, [
+        Requirement([], [CanDrainAQA, HasWaveBeam]),
+        Requirement([], [CanDrainAQA, CanDefeatMediumGeron])
```

Comment:
Formatting is right, in that this is will run and function, but the second array in a Requirement object is a set of ORs. With how you have it written, the item is in logic if you can either drain AQA or if you have Wave Beam or if you can defeat the Geron.

This also tells me I should have commented the Requirement object more on how to use it, as you're not the only one who's been tripped up on this.